### PR TITLE
[FIX] web: correct misaligned headings in TreeView

### LINKED ISSUE #3797: The list view of column headings are not placed parallelly, correctly when the window size is minimized or user has a small computer screen
**The list view of column headings are not placed parallelly, correctly when the window size is minimized or user has a small computer screen**

Impacted versions:
- 7.0 and above

Steps to reproduce:
1. Open any list view with multiple columns.
2. Minimize the window to some extent 
3. then gradually decrease/increase and you'll see):

Current behavior:
- Column headings aren't aligned to each other, are not in one line across each other.

Expected behavior:
- No matter the size of the window, the column headings should be across each other.

_**Note:**_ This problem is not occurring only because of window minimisation: If the size of the computer screen is small, then this problem is reproduced again.

### PR DESCRIPTION
**Pull request by ghost**
If screen were not wide enough for a TreeView, headings in a table were not
aligned to each-other and displayed with a random vertical alignment
instead. The reason of such behavior were arbitrary whitespace around
(within) the column title string.

Closes #3797

Description of the issue/feature this PR addresses:

This PR adrressess #3797, misaligned column headers within TreeView

Current behavior before PR:
- For description of current behavior and an example please refer to #3797
- One more example: 
selection_003

Desired behavior after PR is merged:
- Properly aligned TreeView column titles, as you can see on the following image:
selection_008

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tvibliani** [comment]
It breaks sort symbols however:
selection_005

**Comment by tvibliani** [comment]
I found the origin of extra whitespace that causes misalignment of column titles. It appears QWeb template generating the HTML for table headings is formatted so, that extra whitespace is rendered around the title. Fixing this issue is thus just matter of XML formatting at this point, I'll revert the CSS change I introduced in this PR and change the XML formatting instead, this way issue should be considered as fixed.

**Comment by tvibliani** [comment]
Finally I fixed it without changing any code out there! The only change is XML formatting at the place it caused generation of extra whitespace around title. This PR can be safely merged now, as it fixes the issue it was opened for. Here is a visual effect of the change introduced by the PR:

- Headings before:
selection_010

- Headings after the change:

selection_008

File: addons/web/static/src/xml/base.xml [hunk 1]
```diff
@@ -730,9 +730,9 @@
             </th>
             <t t-foreach="columns" t-as="column">
                 <th t-if="!column.meta and column.invisible !== '1'" t-att-data-id="column.id"
-                    t-attf-class="oe_list_header_#{column.widget or column.type} #{((options.sortable and column.tag !== 'button') ? 'oe_sortable' : null)}"><div>
-                    <t t-if="column.tag !== 'button'"><t t-esc="column.string"/></t>
-                </div></th>
+                    t-attf-class="oe_list_header_#{column.widget or column.type} #{((options.sortable and column.tag !== 'button') ? 'oe_sortable' : null)}">
```
Comment (tvibliani) [inline addons/web/static/src/xml/base.xml]: @ged-odoo what do you think about the matter? Should a value in the above comment preferred over the one already included in the PR?

**Comment by C3POdoo** [comment]
Hello,

This is an automatic message.

Thank you for your contribution but the version 8.0 is no longer supported. We only support the last 3 stable versions so no longer accepts patches into this branch.
We apology if we could not look at your request in time.

If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

Regards,

**Pull request closed without merging**